enviro/plot.py

```diff
@@ -398,74 +393,211 @@ def data_to_table(matrix, var_names):
+
+    method : enviro.views.Method
+        Contains all the information used to create the environmental contour
+        Has among other the attributes method.contour_method and method.return_period
+
+    Returns
+    -------
+    short_file_path_report : string,
+        The path where the pdf, generated based latex, is saved
+        The path continues after 'enviro/static/'
 
-    :param canvas: 
-    :param doc: 
-    :return: 
     """
-    canvas.saveState()
-    #header_content = Image('static/images/ViroVektorCWithDevelopedBy.jpg', width=8 * cm, height=3.819 * cm)
-    header_content = Image('static/images/ViroVektorC.jpg', width=5 * cm, height=1.375 * cm)
-    #footer_content = Image('static/images/Footy2.jpg', width=21 * cm, height=3 * cm)
-    #w, h = footer_content.wrap(doc.width, doc.bottomMargin)
-    #footer_content.drawOn(canvas, 0, 0)
-    w, h = header_content.wrap(doc.width, doc.topMargin)
-    header_content.drawOn(canvas, 8 * cm, doc.height + doc.topMargin - h)
-    canvas.restoreState()
-
-
-def plot_pdf(matrix, user, method_label, probabilistic_model, var_names, var_symbols, method):
+
+    plot_contour(matrix, user, method_label, probabilistic_model, var_names, var_symbols, method)
+    directory_prefix = 'enviro/static/'
+    file_path_contour = directory_prefix + user + '/contour.png'
+    directory_fit_images = directory_prefix + user + '/prob_model/'
+    img_list = os.listdir(directory_fit_images + '/' + str(probabilistic_model.pk))
+
+    latex_content = r"""\section{Results}
+                \subsection{Environmental contour}
+                \includegraphics[width=\textwidth]{""" + file_path_contour + r"""}
+                \subsection{Extreme environmental design conditions}"""\
+                    + get_latex_eedc_table(matrix, var_names, var_symbols) + r"""
+                \section{Methods}
+                \subsection{Associated measurement file}"""
+    if probabilistic_model.measure_file_model:
+        latex_content += r"""file: '\verb|""" + probabilistic_model.measure_file_model.title + r"""|'
+                \subsection{Fitting}"""
+        for img in img_list:
+            img_name = directory_fit_images + str(probabilistic_model.pk) + "/" + img
+            latex_content += r"""\begin{figure}[H]"""
+            latex_content += r"""\includegraphics[width=\textwidth]{""" + img_name + r"""}"""
+            latex_content += r"""\end{figure}"""
+    else:
+        latex_content += r"""No associated file. The model was created by direct input."""
+
+    latex_content += r"""\subsection{Probabilistic model}"""
+
+    # get the equation in latex style
+    dists_model = DistributionModel.objects.filter(probabilistic_model=probabilistic_model)
+    var_symbols = []
+    for dist in dists_model:
+        var_symbols.append(dist.symbol)
+    multivariate_distribution = setup_mul_dist(probabilistic_model)
+    latex_string_list = multivariate_distribution.getPdfAsLatexString(var_symbols)
```

Comment:
pep8 proposes snake_case instead of CamelCase for method names. I propose latex_repr as method name.